src/components/govukComponents/BackLink.js

```diff
@@ -1,11 +1,22 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 
 class BackLink extends Component {
+
   render() {
-    return <div>
-      <p className="govuk-body">Todo: port back link component</p>
-    </div>
+
+    var attributes
+    if (this.props.attributes) {
+      attributes = this.props.attributes
+    }
+
```

Comment:
Lines 8 - 11 could be replaced with a single destructuring assignment:

```const { attributes } = this.props```

Reply:
Having said that, given that you only use it once to spread it into the Link below, it's probably not worth destructuring it anyway? You could just spread like `{...this.props.attributes}`

Reply:
This has been addressed in branch `text-input-v2`